shared/src/androidMain/kotlin/com/intive/picover/shared/di/SharedModule.kt

```diff
@@ -0,0 +1,49 @@
+package com.intive.picover.shared.di
+
+import android.content.Context
+import androidx.work.WorkManager
+import com.intive.picover.articles.repository.ArticlesRepository
+import com.intive.picover.auth.repository.AuthRepository
+import com.intive.picover.images.repository.ImagesRepository
+import com.intive.picover.shared.party.data.repo.PartiesRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import dev.gitlive.firebase.Firebase
+import dev.gitlive.firebase.auth.auth
+import dev.gitlive.firebase.firestore.firestore
+import dev.gitlive.firebase.storage.StorageReference
+import dev.gitlive.firebase.storage.storage
+import kotlinx.coroutines.CoroutineDispatcher
+
+@Module
+@InstallIn(SingletonComponent::class)
+object SharedModule {
+
+	@Provides
+	fun providePartiesRepository() =
+		PartiesRepository(Firebase.firestore)
+
+	@Provides
+	fun provideAuthRepository() =
+		AuthRepository(
+			Firebase.auth,
+			Firebase.storage.reference,
+		)
+
+	@Provides
+	fun provideArticlesRepository(storageReference: StorageReference) =
+		ArticlesRepository(storageReference)
```

Comment:
There an interesting part because `StorageReference` binding is defined in `app:AppModule`